src/EntropyComputation.cpp

```diff
@@ -0,0 +1,116 @@
+/****************************************************************************************
+ *
+ * \file EntropyComputation.cpp
+ * 
+ *
+ * Copyright 2015   Arun Das, University of Waterloo (adas@uwaterloo.ca)
+ *
+ *
+ ****************************************************************************************/
+
+
+
+#include <mcptam/EntropyComputation.h>
+
+//this function evaluates a point's entropy reduction
+double EvaluatePoint(Tracker* tracker, MapPoint& point, KeyFrame& trackerKF, double priorPointCovariance, int pointLevel, double& prevEntropy)
```

Comment:
This is a very big function.  Add an issue to functionalize as we will use this in future code as well.